### PR TITLE
feat: better lobby exceptions handling

### PR DESCRIPTION
**Pull request by LPLafontaineB**
### Description (*)

This PR changes how lobby exceptions are handled. Only some of them will be shown to players through popups.
### Issue Number(s) (*)

Fixes issue(s):
### Manual testing scenarios

1. ...
2. ...
### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by SamuelBellomo** [comment]
any labels?

File: Assets/BossRoom/Scripts/Shared/Net/UnityServices/Lobbies/LobbyServiceFacade.cs [hunk 1]
```diff
@@ -253,28 +253,7 @@ public void RemovePlayerFromLobbyAsync(string uasId, string lobbyId, Action onSu
...
-                        }
-                    }
-
-                    if (!playerFound)
-                    {
-                        Debug.Log($"Player {uasId} has already left the lobby.");
-                    }
-                }
-
+                m_LobbyApiInterface.LeaveLobbyAsync(uasId, lobbyId, onSuccess, onFailure);
```
Comment (SamuelBellomo) [inline Assets/BossRoom/Scripts/Shared/Net/UnityServices/Lobbies/LobbyServiceFacade.cs:256]: what's going on with this change?

**Review by SamuelBellomo — APPROVED** [review]

**Review by fernando-cortez — APPROVED** [review]
This is great. This actually resolves [MTT-2799](https://jira.unity3d.com/browse/MTT-2799) 😄.

**Comment by LPLafontaineB** [comment]
> This is great. This actually resolves [MTT-2799](https://jira.unity3d.com/browse/MTT-2799) 😄.

Oh so it was because of the rate limiting not allowing the host to remove a player from the lobby?

**Comment by fernando-cortez** [comment]
> Oh so it was because of the rate limiting not allowing the host to remove a player from the lobby?

I believe that issue was only pointing out the frequency and presence of the service error popups. The reconnects were still successful.

**Pull request merged**